public/editor/scripts/editor/js/bramble-editor.js

```diff
@@ -14,7 +14,13 @@ define(function(require) {
       // Start loading the Bramble editor resources
       Bramble.load("#webmaker-bramble",{
         url: options.editorUrl,
-        hideUntilReady: true
+        hideUntilReady: true,
+        extensions: {
+          enable: [
+            // Watch and warn users on rename or delete of index.html
+            'bramble-watch-index.html'
```

Comment:
Oh wow! I know what's happening! This is where stuff goes wrong: https://github.com/mozilla/brackets/blob/master/src/bramble/client/main.js#L358

In that line of code, `options.url` is something like `localhost:8000?locale=en-US` because that's what we pass from Thimble as the bramble url. Then, we construct the search params in the API and end up with `?enableExtensions=...&locale=en-US` if a locale was also passed in through the API or if not, `?enableExtensions=...`. So, when we construct the iframe url in the code I linked, we end up with `localhost:8000?locale=en-US?enableExtensions=...` or `localhost:8000?locale=en-US?enableExtensions=...&locale=en-US` which is where the problem lies.